doc/river.1.scd

```diff
@@ -0,0 +1,35 @@
+RIVER(1) "github.com/ifreund/river" "General Commands Manual"
```

Comment:
That is not correct scdoc syntax. It should be something like `river(1) ["github.com/ifreund/river" ["General Commands Manual"]]`

Reply:
The convention of man-pages(7) is to use a capital title, e.g. RIVER not river. The square brackets indicate optional arguments. They are not part of the syntax.